### DISCUSSION TITLE
Github Pages로 개인 블로그 평생 무료로 운영하기 - peterkimzz

### DISCUSSION THREAD

**Discussion by giscus** [peterkimzz/blog#13] [Blog Comments]
# Github Pages로 개인 블로그 평생 무료로 운영하기 - peterkimzz

거의 대부분의 개발자들이 개인 블로그를 운영하라고 얘기한다. 나도 그렇게 생각한다. 왜냐면 분명히 내가 작성했던 코드인데도, 일주일만 지나도 기억이 안나기 때문이다. 그리고 웬만하면 공개해서 작성하라고 하고 싶다. 이미 우리는 누군가가 옛날에 썼던 글을 보고, 문제를 

https://www.peterkimzz.com/github-pages-nuxtjs/

**Comment by peterkimzz** [comment]
블로그가 아니더라도, 개인이나 팀의 라이브러리 사용 문서 사이트도 무료로 만들 수 있어서 알아두면 유용하실거에요.
